manifests/cf-manifest/manifest/050-rds-broker.yml

```diff
@@ -101,6 +79,25 @@ jobs:
                 supportUrl: "https://forums.aws.amazon.com/forum.jspa?forumID=60"
               plan_updateable: true
               plans:
+                - id: "5f2eec8a-0cad-4ab9-b81e-d6adade2fd42"
+                  name: "Free"
+                  description: "5GB Storage, NOT BACKED UP, Dedicated Instance, Max 200 Concurrent Connections. Postgres Version 9.5. DB Instance Class: db.t2.micro."
```

Comment:
max is 87, via `select * from pg_settings where name='max_connections';`
In parameter group, it's LEAST({DBInstanceClassMemory/9531392},5000) and
```
DBInstanceClassMemory
Returns the number of bytes of memory allocated to the DB instance class associated with the current DB instance, less the memory used by the Amazon RDS processes that manage the instance.
```
87 thus implies the DBInstanceClassMemory is 830 MiB. Meaning about 244MiB is reserved for the system.